alcs-api/src/application/application-decision/application-decision.service.ts

```diff
@@ -95,30 +106,40 @@ export class ApplicationDecisionService {
       );
     }
 
```

Comment:
I wonder if it is possible to break this method into smaller parts

Reply:
I cleaned up the check at the top but couldnt find any clean way to split up the logic.